### ISSUE TITLE
Website http://sinonjs.org/, in examples,`throttle` should be called `debounce`

### ISSUE THREAD

**Issue by ALeschinsky** [sinonjs/sinon#1605]
```
function throttle(callback) {
    var timer;
    return function () {
        clearTimeout(timer);
        var args = [].slice.call(arguments);
        timer = setTimeout(function () {
            callback.apply(this, args);
        }, 100);
    };
}
```
I'm pretty sure that's debounce... throttle should call the function immediately, this one waits first...

**Comment by fatso83** [comment]
The docs live here in the report. Under /docs. You can actually do the entire fix, including the PR, without leaving your browser, using the in-browser editing of Github. Feel free ;-)

**Comment by ALeschinsky** [comment]
PR: https://github.com/sinonjs/sinon/pull/1672/files

**Comment by mroderick** [comment]
This was fixed with #1672
